lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -191,17 +189,7 @@ class AwsCompileFunctions {
         newVersionObject);
     }
 
-    // Add function to Outputs section
-    const newOutput = this.cfOutputDescriptionTemplate();
-    newOutput.Value = { 'Fn::GetAtt': [functionLogicalId, 'Arn'] };
-
-    const newOutputObject = {
-      [functionOutputLogicalId]: newOutput,
-    };
-
-    _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs,
-      newOutputObject);
-
+    // Add function versions to Outputs section
     const functionVersionOutputLogicalId = this.provider.naming
```

Comment:
Yes, that's right. The conditional for adding it to the template (or not) is [here](https://github.com/serverless/serverless/pull/3125/files#diff-eb4669139d74284558dc965accdd94ceR199)